This symbol is already defined in Xt. This change matches the code in Athena.
--- lib/Xm/VendorS.c.orig	2017-08-31 07:12:38
+++ lib/Xm/VendorS.c	2024-05-14 17:03:27
@@ -529,8 +529,10 @@
     }                            
 };	   
 
+#if !defined(__APPLE__)
 externaldef(vendorshellwidgetclass) WidgetClass 
   vendorShellWidgetClass = (WidgetClass) (&vendorShellClassRec);
+#endif
 
 
 
